src/views/PlaygroundView.tsx

```diff
@@ -39,3 +40,26 @@ const LinksContainer = styled.div`
 `
 
 export default PlaygroundView
+
+const VideoMetadata = () => {
+  const { setFile, metadata } = useVideoMetadata()
+  console.log({ metadata })
+  return (
+    <div>
+      <input
+        type="file"
+        accept="video/*"
```

Comment:
```suggestion
        accept="video/*,.mkv"
```

It seems `.mkv` files don't have a MIME type? Something along those lines

Reply:
@eldiegod this wasn't resolved